graphql/schema.graphql

```diff
@@ -84,7 +84,8 @@ type Mutation {
   schedulePatch(patchId: String!, configure: PatchConfigure!): Patch!
   schedulePatchTasks(patchId: String!): String
   unschedulePatchTasks(patchId: String!, abort: Boolean!): String
-  restartPatch(patchId: String!, abort: Boolean!, taskIds: [String!]!): String
+  restartVersions(patchId: String!, abort: Boolean!, versionsToRestart: [VersionToRestart!]!): String
```

Comment:
This is a good opportunity to update the apollo cache. If we return a `Version` or `[Version]` we can include the Version updates along with all the tasks status updates In the response and avoid an extra request on the front end to refetch fresh data.

Reply:
The best way to do this would be to edit ModifyVersionHandler to return the version so that it could be returned here. Since that is a bigish refactor because the function is pretty nested for different actions, and this ticket is holding up the release of the project, how would you feel about creating a ticket for this and doing it at a later point?

Reply:
We can do either way. But It would be easier to implement this prior to the release since that will avoid any backwards compatibility issues that may come with updating this route later on Since this route is not yet used by anything. 
Could we simply do a findVersionById() here and return the versions. https://github.com/evergreen-ci/evergreen/pull/4999/files#diff-a3ce50f20b746b196f29c6d414e685b0110c24b768bcf83756de8d842c98f6dcR2189
it would accomplish the same effect and would not require refactoring any functions.

Reply:
Aren't we doing this to avoid a db call though?

Reply:
did it

Reply:
Thanks! For clarity we will be making the db call either way. But doing it this way allows us to avoid an extra network call to fetch fresh data, Since this will return fresh data as part of the initial response.